docs/howto_config.md

```diff
@@ -10,48 +10,55 @@ The MicroShift configuration file must be located at `~/.microshift/config.yaml`
-| serviceNodePortRange| --service-node-port-range | MICROSHIFT_CLUSTER_SERVICENODEPORTRANGE | The port range allowed for Kubernetes services of type NodePort (WARNING: see the dedicated section)
-| domain              | --cluster-domain          | MICROSHIFT_CLUSTER_DOMAIN               | Base DNS domain used to construct fully qualified pod and service domain names
-| url                 | --url                     | MICROSHIFT_CLUSTER_URL                  | URL of the API server for the cluster.
-| nodeIP              | --node-ip                 | MICROSHIFT_NODEIP                       | The IP address of the node, defaults to IP of the default route
-| nodeName            | --node-name               | MICROSHIFT_NODENAME                     | The name of the node, defaults to hostname
+| Field Name            | CLI Argument              | Environment Variable                    | Description |
+|-----------------------|---------------------------|-----------------------------------------|-------------|
+| cidr (clusterNetwork) | --cluster-cidr            | MICROSHIFT_CLUSTER_CLUSTERCIDR          | A block of IP addresses from which Pod IP addresses are allocated
+| serviceNetwork        | --service-cidr            | MICROSHIFT_CLUSTER_SERVICECIDR          | A block of virtual IP addresses for Kubernetes services
+| serviceNodePortRange  | --service-node-port-range | MICROSHIFT_CLUSTER_SERVICENODEPORTRANGE | The port range allowed for Kubernetes services of type NodePort
+| baseDomain            | --cluster-domain          | MICROSHIFT_CLUSTER_DOMAIN               | Base DNS domain used to construct fully qualified pod and service domain names
+| url                   | --url                     | MICROSHIFT_CLUSTER_URL                  | URL of the API server for the cluster.
+| nodeIP                | --node-ip                 | MICROSHIFT_NODEIP                       | The IP address of the node, defaults to IP of the default route
+| nodeName              | --node-name               | MICROSHIFT_NODENAME                     | The name of the node, defaults to hostname
+| logLevel              | --v                       | MICROSHIFT_LOGVLEVEL                    | Log verbosity (Normal, Debug, Trace, TraceAll)
 | subjectAltNames     | --subject-alt-names       | MICROSHIFT_SUBJECTALTNAMES              | Subject Alternative Names for apiserver certificates
-| logVLevel           | --v                       | MICROSHIFT_LOGVLEVEL                    | Log verbosity (0-5)
 
 ## Default Settings
 
 In case `config.yaml` is not provided, the following default settings will be used.
 
 ```yaml
-cluster:
-  clusterCIDR: 10.42.0.0/16
-  serviceCIDR: 10.43.0.0/16
-  serviceNodePortRange: 30000-32767
-  domain: cluster.local
-  url: https://127.0.0.1:6443
-nodeIP: ""
 nodeName: ""
-subjectAltNames: nil
-logVLevel: 0
+nodeIP: ""
+url: https://127.0.0.1:6443
+network:
+  clusterNetwork:
+    - cidr: 10.42.0.0/16
+  serviceNetwork:
+    - 10.43.0.0/16
```

Comment:
@mangelajo that's because it's defined this way in OpenShift APIs (see https://github.com/openshift/api/blob/master/config/v1/types_network.go#L89-L98) and we're aligning with that.

Reply:
oh, right. We are just inheriting it.